fix build errors

cleverca22 opened the pull request.

cleverca22 committed:
fix build errors

avieth commented on src/exec/Shelley.hs:
```diff
@@ -5,11 +5,13 @@
 module Shelley where
 
 import qualified Codec.CBOR.Read as CBOR (DeserialiseFailure)
+import qualified Codec.Serialise as Serialise (decode, encode)
```
I don't think these are needed.

avieth commented on src/exec/Shelley.hs:
```diff
@@ -5,11 +5,13 @@
 import Data.Void (Void)
 import Network.Socket (SockAddr)
 
+import Cardano.Binary (fromCBOR, toCBOR)
```
I already added these back in the base PR 044a1052046358947fc4062801f50b713e06d17d

cleverca22 committed:
regen nix files
set stack.yaml to the tracer version of ouroboros-network
remove validator tests
